glean-core/src/system.rs

```diff
@@ -79,3 +81,31 @@ pub const OS: &str = "Solaris";
     target_os = "solaris",
 )))]
 pub const OS: &str = "unknown";
+
+#[cfg(any(target_os = "macos", target_os = "windows"))]
+/// Returns Darwin kernel version for MacOS, or NT Kernel version for Windows
```

Comment:
Hm, I guess this is really the only file where we have cfg before the comment.
I was about to suggest to swap it around, but let's not do it here now.